pmm-app/tests/ia/ruleTemplates_test.js

```diff
@@ -0,0 +1,116 @@
+const assert = require('assert');
+const page = require('./pages/ruleTemplatesPage');
+
+const templates = new DataTable(['path']);
+
+for (const i of Object.values(page.ruleTemplate.paths)) {
+  templates.add([i]);
+}
+
+Feature('IA: Alert rule templates');
+
+Before(async (I, ruleTemplatesPage) => {
+  I.Authorize();
+  ruleTemplatesPage.openRuleTemplatesTab();
+});
+
+Scenario(
+  'PMM-T510 Verify built-in rule templates are non-editable @ia @not-pr-pipeline',
+  async (I, ruleTemplatesPage) => {
+    const editButton = ruleTemplatesPage.buttons
+      .editButtonBySource(ruleTemplatesPage.templateSources.builtin);
+
+    I.waitForVisible(editButton, 30);
+    I.seeAttributesOnElements(editButton, { disabled: true });
+  },
+);
+
+Scenario(
+  'Verify rule templates list elements @ia @not-pr-pipeline',
+  async (I, ruleTemplatesPage) => {
+    ruleTemplatesPage.columnHeaders.forEach((header) => {
+      const columnHeader = ruleTemplatesPage.elements.columnHeaderLocator(header);
+
+      I.waitForVisible(columnHeader, 30);
+    });
+    const templateName = await I.grabTextFrom(ruleTemplatesPage.elements.templateName);
+
+    templateName.forEach((name) => {
+      assert.ok(name.length > 0, 'Rule Template name should not be empty');
+    });
+    I.seeElement(ruleTemplatesPage.buttons.openAddTemplateModal);
+  },
+);
+
+Scenario(
+  'Add rule template modal elements @ia @not-pr-pipeline',
+  async (I, ruleTemplatesPage) => {
+    I.click(ruleTemplatesPage.buttons.openAddTemplateModal);
+    I.see(ruleTemplatesPage.messages.modalHeaderText, ruleTemplatesPage.elements.modalHeader);
+    I.seeElement(ruleTemplatesPage.buttons.closeModal);
+    I.seeElement(ruleTemplatesPage.buttons.uploadFile);
+    I.seeElement(ruleTemplatesPage.buttons.addTemplate);
+    I.seeElement(ruleTemplatesPage.buttons.cancelAdding);
+  },
+);
+
+Scenario(
+  'PMM-T500 Add rule templates @ia @not-pr-pipeline',
+  async (I, ruleTemplatesPage) => {
+    const [templateName, fileContent] = ruleTemplatesPage.ruleTemplate
+      .templateNameAndContent(ruleTemplatesPage.ruleTemplate.inputFilePath);
+    const expectedSourceLocator = ruleTemplatesPage
+      .getSourceLocator(templateName, ruleTemplatesPage.templateSources.ui);
+    const editButton = ruleTemplatesPage.buttons
+      .editButtonBySource(ruleTemplatesPage.templateSources.ui);
+
+    I.click(ruleTemplatesPage.buttons.openAddTemplateModal);
+    I.fillField(ruleTemplatesPage.fields.templateInput, fileContent);
+    I.click(ruleTemplatesPage.buttons.addTemplate);
+    ruleTemplatesPage.verifyPopUpMessage(ruleTemplatesPage.messages.successfullyAdded);
+    I.waitForVisible(expectedSourceLocator, 30);
+    I.seeAttributesOnElements(editButton, { disabled: null });
+  },
+);
+
+Data(templates)
+  .Scenario(
+    'PMM-T482 PMM-T499 Upload rule templates @ia @not-pr-pipeline',
```

Comment:
where/when you want to run these tests?

Reply:
I do not want to run these tests so far. Should I add some specific tag for it?

Reply:
I think the current tags will be fine to skip it from all jobs, I don't think we have a job without tags, so this should be fine.